doc/api/cli.md

```diff
@@ -944,11 +944,15 @@ for the very first unhandled rejection in case no [`unhandledRejection`][] hook
 is used.
 
 Using this flag allows to change what should happen when an unhandled rejection
-occurs. One of three modes can be chosen:
+occurs. One of the following modes can be chosen:
 
+* `throw`: Emit [`unhandledRejection`][]. If this hook is not set, raise the
+  unhandled rejection as an uncaught exception.
 * `strict`: Raise the unhandled rejection as an uncaught exception.
 * `warn`: Always trigger a warning, no matter if the [`unhandledRejection`][]
   hook is set or not but do not print the deprecation warning.
+* `warn-with-error`: Emit [`unhandledRejection`][]. If this hook is not set,
```

Comment:
What about calling this `warn-with-error-code`? Otherwise I'd expect "error" to relate to an exception.

Reply:
better `exit` instead of `error` - `warn-with-exitCode`

Reply:
I prefer `warn-with-error-code` over `warn-with-exit-code`. We always exit with an exit code (0, hopefully!) but the process should only exit with a non-zero "error code" if an error occurred.